src/databricks/labs/ucx/assessment/pipelines.py

```diff
@@ -50,6 +50,14 @@ def _assess_pipelines(self, all_pipelines) -> Iterable[PipelineInfo]:
             pipeline_config = pipeline_response.spec.configuration
             if pipeline_config:
                 failures.extend(self.check_spark_conf(pipeline_config, "pipeline"))
+            pipeline_cluster = pipeline_response.spec.clusters[0]
```

Comment:
Can you iterate over clusters instead of picking up the first?

Reply:
Changed the code to iterate through the cluster.